File: JavaScript/프로그래머스/0/120845. 주사위의 개수/주사위의 개수.js
function solution(box, n) {
    var answer = 0;
    var a = Math.floor(box[0]/n);
    var b = Math.floor(box[1]/n);
    var c = Math.floor(box[2]/n);
    var answer = a*b*c;
    return answer;
}

File: 프로그래머스/0/120845. 주사위의 개수/주사위의 개수.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// box_len은 배열 box의 길이입니다.
int solution(int box[], size_t box_len, int n) {
    int answer = 0;
    int a = box[0]/n;
    int b = box[1]/n;
    int c = box[2]/n;
    answer = a*b*c;
    return answer;
}